build.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python
+
+# Keep this file compatible with Python 2 -- it's the first file users will run.
+# Even if we end up requiring a different version for the rest of the project,
+# this file should probably not crash and burn.
+
+import argparse
+import errno
+import itertools
+import os
+import subprocess
+import sys
+import traceback
+
+try: import ctypes  # necessary for Windows
```

Comment:
This shouldn't be needed, can we just use builtin python unicode routines?

Reply:
I'm using this to call a platform API though?